runtime/src/snapshot_utils.rs

```diff
@@ -395,6 +396,25 @@ pub fn archive_snapshot_package(
         ("duration_ms", timer.as_ms(), i64),
         ("size", metadata.len(), i64)
     );
+
+    info!("Create snapshot local file in {:?}", tar_dir);
+    create_snapshot_local_file(tar_dir)?;
+    Ok(())
+}
+
+pub fn has_snapshot_local_file(snapshot_archive_dir: &Path) -> bool {
```

Comment:
Please use `impl AsRef<Path>` instead of `&Path`. This allows both `Path` and `PathBuf` to be used.
(and in the other locations)

I.e. `snapshot_archives_dir: impl AsRef<Path>`